misc/AssemblyInfo/src/Test/project.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Hello title",
+  "description": "Hello description",
+  "language": "en",
+  "copyright": "Hello copyright",
```

Comment:
You may want to add some unicode code point here. For example U+00A9 is commonly used for copy right sign.